### PR TITLE
adding alpha-lambda and bunyan

### PR DESCRIPTION
**Pull request by jcdietrich**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "eslint . && npx ava"
+    "test": "eslint . && nyc ava"
```
Comment (AntonBazhal) [inline package.json]: I'd have this as separate scripts, e.g `lint`, `test:unit`, and `test`

File: src/handlers/assignLearning.test.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 require('must/register');
 const lambdaTester = require('lambda-tester');
-const handler = require('./assignLearning.js').execute;
+const handler = require('./assignLearning.js');
```
Comment (AntonBazhal) [inline src/handlers/assignLearning.test.js]: I like to have all the imports separated into two groups: libraries and modules, with alphabetical sorting inside each.

File: src/utils/lambdaHandler.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+const bunyan = require('alpha-lambda-bunyan');
+
+module.exports = function handlerWrap(handler) {
+    return lambdaHandler()
+        .use(bunyan())
+        .use((event, context, next) => {
+            context.log.info(event, 'incoming event');
+            return next();
+        })
+        .use(handler);
```
Comment (AntonBazhal) [inline src/utils/lambdaHandler.js]: I'd definitely add an error handler to log errors, at least.

File: package.json [hunk 4]
```diff
@@ -14,6 +14,11 @@
     "eslint-config-airbnb-base": "^12.1.0",
     "eslint-plugin-import": "^2.11.0",
     "lambda-tester": "^3.4.0",
-    "must": "^0.13.4"
+    "must": "^0.13.4",
```
Comment (AntonBazhal) [inline package.json]: What's the reason to use `must` instead of `chai`?

Comment (dougmoscrop) [reply package.json]: https://github.com/moll/js-must#beware-of-libraries-that-assert-on-property-access

File: package.json (hunk 1 above)
Comment (dougmoscrop) [reply package.json]: yeah, and "posttest" can "npm run lint"

File: src/utils/lambdaHandler.js (hunk 3 above)
Comment (dougmoscrop) [reply src/utils/lambdaHandler.js]: Let's wait a second. I am moving stuff to shared lib. I will work with JCD on this.

File: package.json (hunk 4 above)
Comment (AntonBazhal) [reply package.json]: Saw it, but `expect` does not have this issue.

**Pull request merged**